### ISSUE TITLE
[Accessibility]Form control fields should be nested between <FORM> and </FORM> tags.

### ISSUE THREAD

**Issue by atreyeemukhopadhyay** [kimchi-project/kimchi#907]
Fieldset/Label/Legend Tags must be nested between <FORM> and </FORM> tags.
